service/key/key.go

```diff
@@ -88,8 +88,10 @@ func GetServicePriority(cluster metav1.Object) string {
 }
 
 func GetOrganization(cluster metav1.Object) string {
-	if organization, ok := cluster.GetLabels()[OrganizationLabel]; ok && organization != "" {
-		return organization
+	re := regexp.MustCompile(`org-(.*)`)
+	match := re.FindStringSubmatch(cluster.GetNamespace())
+	if len(match) >= 2 {
+		return match[1]
```

Comment:
Could we fetch the organization from the namespace instead? In that case, if we ever change the name format used for our organizations (currently `org-$orgName`) we wouldn't need to change it here

There should be a label in the namespace with the organization name called `"giantswarm.io/organization"`.

Reply:
We there should bé a label, are you sure it's set everywhere ? Because if it is, we can use thé label on thé namespace for sure.

Reply:
fetching it from namespace, require a request to the api, 
You think that's okay, from a performance perspective...

Reply:
I'm fine with 1 call to the api for each cluster every 5 minutes yes :)

Reply:
But are we sure the label is set?

Reply:
it should be set, by the organization-operator

Reply:
Can you confirm this is the case? :)

Reply:
https://github.com/giantswarm/organization-operator/blob/main/service/controller/resource/organization/resource.go#L78